low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -0,0 +1,60 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import subprocess
+
+from cortx.sspl.bin.sspl_constants import PRODUCT_FAMILY
+from cortx.sspl.lowlevel.framework.utils.utility import Utility
+
+TEST_DIR = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/sspl_test"
+
+class SSPLTestCmd:
+    "Run SSPL Test"
+    name = "sspl-test"
+
+    def __init__(self, args: list):
+        self.args = args
+    
+    def process(self):
+        output=None
+        print("Processing %s %s" %(self.name, self.args))
+        CMD="rpm -qa | grep sspl-test"
```

Comment:
Need to use rpm validator functions from cortx-utils eventually, expected to be available tomorrow.

https://github.com/Seagate/cortx-utils/pull/162/files

Reply:
Done.

Reply:
I dont see rpm validators being used from cortx-utils

Reply:
Changes are done